### ISSUE TITLE
devcontainer shell warning when using xdeb

### ISSUE THREAD

**Issue by thetredev** [thetredev/xdeb-install#13]
`xdeb` version: current master: https://github.com/toluschr/xdeb/tree/c61c12ec06f90a6ec67dfa1e4631d1714b0483f1

Within the devcontainer, running
```
$ go run main.go deb <package>
```
works, but running `xdeb` prints the following warning a couple times:
```
/usr/local/bin/xdeb: line 59: warning: command substitution: ignored null byte in input
```

This does not affect non-container usage. Probably has to do with the base image which is primarily used for CI environments and has other XBPS repositories enabled.

**Comment by thetredev** [comment]
Using default XBPS repositories doesn't fix the problem either.

**Comment by thetredev** [comment]
Running
```
$ xdeb -Sde <deb file>
```
manually results in the same warnings.

**Comment by thetredev** [comment]
Found the issue: It was because we were using the `voidlinux-buildroot` base image. Switching to `ghcr.io/void-linux/void-glibc-full:20231003R1` did the trick.

**Comment by thetredev** [comment]
Fixed with 43bde70
